CHANGELOG.rst

```diff
@@ -5,6 +5,22 @@ CHANGELOG
 Unreleased
 ==========
 
+2.12.0
+==========
+* Updated `README` with Oversampling Mitigation examples `PR368 https://github.com/aws/aws-xray-sdk-python/pull/368`
```

Comment:
nit: this is probably not necessary to include in the changelog since this change has been reflected in the ReadME (not linked to the release)

Reply:
Updated!